runtime/fundamentals/linting_and_formatting.mdx

```diff
@@ -103,21 +103,118 @@ jobs:
-| use-tabs           | Use tabs instead of spaces for indentation             | **false**  | true, false             |
+#### `bracePosition`
+Define brace position for blocks
+- **Default:** `sameLine`
+- **Possible values:** `maintain`, `sameLine`, `nextLine`, `sameLineUnlessHanging`
+
+#### `jsx.bracketPosition`
+Define bracket position for JSX
+- **Default:** `nextLine`
+- **Possible values:** `maintain`, `sameLine`, `nextLine`
+
+#### `jsx.forceNewLinesSurroundingContent`
+Forces newlines surrounding the content of JSX elements
+- **Default:** `false`
+- **Possible values:** `true`, `false`
+
+#### `jsx.multiLineParens`
+Surrounds the top-most JSX element or fragment in parentheses when it spans multiple lines
+- **Default:** `prefer`
+- **Possible values:** `never`, `prefer`, `always`
+
+#### `indent-width`
+Define indentation width
+- **Default:** `2`
+- **Possible values:** `number`
+
+#### `line-width`
+Define maximum line width
+- **Default:** `80`
+- **Possible values:** `number`
+
+#### `newLineKind`
+Thee newline character to use
+- **Default:** `lf`
+- **Possible values:** `auto`, `crlf`, `lf`, `system`
+
+#### `nextControlFlowPosition`
+Define position of next control flow
+- **Default:** `sameLine`
+- **Possible values:** `sameLine`, `nextLine`, `maintain`
+
+#### `no-semicolons`
+Don't use semicolons except where necessary
+- **Default:** `false`
+- **Possible values:** `true`, `false`
+
+#### `operatorPosition`
+Where to place the operator for expressions that span multiple lines
+- **Default:** `sameLine`
+- **Possible values:** `sameLine`, `nextLine`, `maintain`
+
+#### `prose-wrap`
+Define how prose should be wrapped
+- **Default:** `always`
+- **Possible values:** `always`, `never`, `preserve`
+
+#### `quoteProps`
+Control quoting of object properties
+- **Default:** `asNeeded`
+- **Possible values:** `asNeeded`, `consistent`, `preserve`
+
+#### `singleBodyPosition`
+The position of the body in single body blocks
+- **Default:** `sameLineUnlessHanging`
+- **Possible values:** `sameLine`, `nextLine`, `maintain`, `sameLineUnlessHanging`
+
+#### `single-quote`
+Use single quotes
+- **Default:** `false`
+- **Possible values:** `true`, `false`
+
+#### `spaceAround`
+Control spacing around enclosed expressions
+- **Default:** `false`
+- **Possible values:** `true`, `false`
+
+#### `spaceSurroundingProperties`
+Control spacing surrounding single line object-like nodes
+- **Default:** `true`
+- **Possible values:** `true`, `false`
+
+#### `trailingCommas`
+Control trailing commas in multi-line arrays/objects
+- **Default:** `always`
+- **Possible values:** `always`, `never`
+
+#### `typeLiteral.separatorKind`
+Define separator kind for type literals
+- **Default:** `semiColon`
+- **Possible values:** `comma`, `semiColon`
+
+#### unstable-component
+Enable formatting Svelte, Vue, Astro and Angular files
+
+#### `unstable-sql`
+Enable formatting SQL files
```

Comment:
I think these are configured via the `"unstable": [...]` config. Maybe we should make that clearer?

Reply:
Good spot. ok I'm noting that in an issue in order to expedite this one which feels like a glaring omission that needs immediate attention